src/components/components/EditNewsForm.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import apiPostagem from "../../services/apiPostagem";
+import { withRouter } from "react-router-dom";
+import "./editNewsForm.css";
+import apiNoticias from "../../services/apiNoticias";
+
+
+
+class EditNewsForm extends React.Component {
+
+    constructor(props){
+      super(props);
+      this.state = {
+        title: "",
+        subtitle: "",
+        text:  "",
+        image1: "",
+        image2: "",
+        image3: "",
+        post_id: "",
+        posts: []
+      };
+
+      this.id = this.props.match.params.newsId;
+
+      // this.history = useHistory();
+  
+      this.handleChange = this.handleChange.bind(this);
+      this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    async getNews(){
+      const response = await apiNoticias.get("news/"+this.id);
+      const posts = await apiPostagem.get(`posts?limit=100&page=0`);
+      this.setState(response.data);
+      this.setState({posts: posts.data.rows});
+    }
+
+    componentDidMount(){
+      this.getNews();
+    }
+  
+    handleChange(event){
```

Comment:
Codacy found an issue: [Method 'handleChange' has a complexity of 13. Maximum allowed is 4.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6404581)